src/audio/wiiu/SDL_wiiuaudio.c

```diff
@@ -427,7 +499,7 @@ static Uint8* WIIUAUDIO_GetDeviceBuf(_THIS) {
 }
 
 static void WIIUAUDIO_CloseDevice(_THIS) {
-    if (AXIsInit()) {
```

Comment:
I'd probably make the AX (de-)initialization completely independent from opening and closing devices, by moving it into `WIIUAUDIO_Init` and the `Deinitialize` callback.